### ISSUE TITLE
Not working on raspberry pi (wrong ELF class)

### ISSUE THREAD

**Issue by giejay** [tejado/pokemongo-api-demo#190]
Could someone create a cross compiled libencrypt.so for ARM? Or is this already available? 

Currently I'm looking at these errors: libencrypt.so: wrong ELF class: ELFCLASS64

**Comment by ProLoDs** [comment]
raspberry is not an AMDx64 cpu its arm... you need to compile the lib on the raspberry again

**Comment by giejay** [comment]
Yeah, I understand that but I dont have any experience with compiling so I wondered if someone already did this

**Comment by ProLoDs** [comment]
go in the src folder: `gcc -shared -fPIC encrypt.c -o libencrypt.so -std=c11`

**Comment by zvvyt** [comment]
@giejay https://www.reddit.com/r/pokemongodev/comments/4wjlq5/linux_arm_build/?
Though now I'm stuck at the "pogom/libencrypt.so: undefined symbol: encrypt"-error..

**Comment by giejay** [comment]
The owner renamed the method encrypt to encryptMethod in the latest commit. Therefore I checked out the commit before that one (git checkout 8ecc80689dceaf6b8e68e9d304634e1729f190bf) and it works now:)
